src/main/scala/nodes/util/Densify.scala

```diff
@@ -0,0 +1,21 @@
+package nodes.util
+
+import breeze.linalg.DenseVector
+import breeze.linalg.Vector
+import workflow.Transformer
+
+/**
+ * Transformer to densify vectors into DenseVectors.
+ */
+case class Densify[T <: Vector[Double]]() extends Transformer[T, DenseVector[Double]] {
```

Comment:
I think this is a fine solution - basically makes this a no-op if it can be, and densify it if necessary.